android/app/build.gradle

```diff
@@ -101,14 +101,25 @@ android {
     buildToolsVersion '25.0.1'
```

Comment:
@seland can you please have a very close look at the changes? I don't like to change the gradle files.

Reply:
@seland ping.